src/presets/alwaysdata.ts

```diff
@@ -0,0 +1,8 @@
+import { defineNitroPreset } from "../preset";
+
+export const alwaysdata = defineNitroPreset({
+  extends: "node-server",
+  commands : {
+    deploy: "scp -r . [account].alwaysdata.net:~/www/"
```

Comment:
(you can try locally, with `./` nitro smartly generated actual output directory name!)

```suggestion
    deploy: "scp -r ./ [account].alwaysdata.net:~/www/"
```

Reply:
Ok, didn't notice that, my bad. I moved it to using rsync for more clarity and better compatibility.